index.html

```diff
@@ -35,14 +41,17 @@ <h6 class="connect">let's connect</h6>
                 <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
                 <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
             </ul>
-        </main>
+            </main>
+        </div>
     </div>
     <section class="page-development">
         <div class="works">
             <img src="img/Snapshoot Portfolio.png" class="snapshoot">
+            <div class="photo-one"></div>
             <div class="left-block">
                 <div class="primary-text-block">
                     <h2 class="project-title">Tonic</h2>
+                    <h2 class="project-title-desktop">tonic</h2>
```

Comment:
- [ ] The info about the work should be in one line. The `Back End Dev` text is currently breaking. Kindly fix it in one line. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![backend_design](https://user-images.githubusercontent.com/70483835/221088613-da140c12-7afa-4a5d-a413-65d4ff0d688e.png) | ![figma_backend_dev](https://user-images.githubusercontent.com/70483835/221088591-7719d579-9df4-4189-9346-86887c7896b5.png)

- [ ] The background color of the page doesn't match the Figma design. Kindly fix it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![background_color_design](https://user-images.githubusercontent.com/70483835/221089278-8cf4583f-69a7-4c5c-86af-eb678baf0cf5.png) | ![background_figma](https://user-images.githubusercontent.com/70483835/221089264-471b5c9a-a1a5-4098-8a92-80fe56f42fdb.png)

- [ ] The `See Project` button that takes you to the Live Demo of the project or to the GitHub repository and also tries to match the Figma design as well. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![see_design](https://user-images.githubusercontent.com/70483835/221089738-00b27dc2-f811-453b-8c22-5871bec02ec7.png) | ![figma_see](https://user-images.githubusercontent.com/70483835/221089568-5b084ef0-50e5-45f3-a51b-8fee809a8cad.png)

Reply:
@mahbubul14 hi! I have a question on which screen size did you saw that the back end dev line broke? Because I’m checking and in 1440px it’s okay. Should I change it anyway?

Reply:
Hi, @federicaulzurrun since I see that you asked the previous reviewer, the back-end dev line breaks even at `1440px` and more, it is most likely an issue with the amount of width attributed to the container because it's present in all cards. 👇🏽 
![Screenshot 2023-02-24 at 08 23 29](https://user-images.githubusercontent.com/45739963/221189340-f5676b6b-e871-4a77-b904-d5971a0bb2d1.png)

Also, another thing I noticed is that there's some sort of white background behind the cards when resizing, it starts below `1440px` and goes on. See below 👇🏽  
![Screenshot 2023-02-24 at 08 32 04](https://user-images.githubusercontent.com/45739963/221191068-37018a09-8f86-4f15-bed9-1561fb21aebe.png)